playwright/tests/admin/plugins.spec.ts

```diff
@@ -287,17 +290,26 @@ test.describe(() => {
     await page.locator('[data-test=script-runner-file]').click()
     await page.locator('text=Save File').click()
     await expect(page.locator('text=File Save As')).toBeVisible()
-    const pwtest = page.getByText('PW_TEST')
     await page
-      .locator('.v-list-item')
-      .filter({ has: pwtest })
-      .locator('.v-list-item-action')
+      .locator('.v-list-group:has-text("PW_TEST")')
+      .first()
+      .getByRole('button')
       .click()
-    await page.getByLabel('PW_TEST').getByText('procedures').click()
-    const prepend = await page.locator('[data-test=file-open-save-filename] input').inputValue()
-    await page.locator('[data-test=file-open-save-filename] input').fill(`${prepend}/save_new.rb`)
+    await page
+      .locator('.v-list-group:has-text("PW_TEST")')
+      .first()
+      .locator('.v-list-item:has-text("procedures")')
+      .click()
+    const prepend = await page
+      .locator('[data-test=file-open-save-filename] input')
+      .inputValue()
+    await page
+      .locator('[data-test=file-open-save-filename] input')
+      .fill(`${prepend}/save_new.rb`)
```

Comment:
This section of playwright was failing so I took a similar example from the `file-menu.spec.ts`